src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -114,6 +115,9 @@ MapDocumentActionHandler::MapDocumentActionHandler(QObject *parent)
     mActionSelectNextLayer = new QAction(this);
     mActionSelectNextLayer->setShortcut(Qt::CTRL + Qt::Key_PageUp);
 
+    mActionSelectAllLayers = new QAction(this);
+    mActionSelectAllLayers->setShortcut((Qt::CTRL) + Qt::Key_B);
```

Comment:
Why did you choose <kbd>Ctrl+B</kbd>? Did you check if there's an equivalent shortcut in other applications with layers, like Krita, GIMP or Photoshop?

Reply:
I wasn't sure what to pick, but checking similar apps is a great idea.  It looks like Krita and GIMP do not have shortcuts for this, but Photoshop uses Ctrl+Alt+A.  How does that sound?

Reply:
I considered <kbd>Ctrl+Alt+A</kbd> as well, so let's go with that!